datafusion/common/src/config.rs

```diff
@@ -252,6 +252,11 @@ config_namespace! {
         /// string length and thus DataFusion can not enforce such limits.
         pub support_varchar_with_length: bool, default = true
 
+       /// If true, `VARCHAR` is mapped to `Utf8View` during SQL planning.
+       /// If false, `VARCHAR` is mapped to `Utf8`  during SQL planning.
+       /// Default is false.
+        pub default_varchar_views: bool, default = false
```

Comment:
Could we define this parameter more clearly and readably, such as `map_varchar_to_utf8view`?

Reply:
> Could we define this parameter more clearly and readably, such as `map_varchar_to_utf8view`?

Thank you @Weijun-H for review and good suggestion! Addressed in latest PR.